### ISSUE TITLE
Call GET Function

### ISSUE THREAD

**Issue by cheasan** [roccomuso/node-webhooks#15]
Correct me if m wrong, I saw that the function accepts only POST request. 
I wonder if there is a way to specify URL method.

**Comment by roccomuso** [comment]
You're right ([index.js#L107](https://github.com/roccomuso/node-webhooks/blob/master/index.js#L107)).
In the current version there's no way to specify a different HTTP method.
